db/migrate/20190702222417_create_lti_configurations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLtiConfigurations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :lti_configurations do |t|
+      t.text :consumer_key, null: false
```

Comment:
I think we will actually also want to ensure this is unique, as according to the [Security Specification](http://ltiapps.net/guide/LTI_Best_Practice.pdf) best practices dictate that the `oauth_consumer_key` is a "unique identifier for the tool consumer"

Reply:
Yes, that is something we should add. I'll also make sure the `lms_link` is unique too since we don't want teachers to be linking a LMS classroom that was already added.

Reply:
I think actually the link itself should be fine to be duplicated, it's only what we redirect professors to to make their lives easier, and there's no guarantee the URL on an LMS will be unique for classes (Sakai, my school's LMS, for example operates exclusively through POST requests, so there is essentially only one URL visible in the address bar at all times)